data/json/snippets/fliers.json

```diff
@@ -261,19 +261,19 @@
       },
       {
         "id": "flier_65",
-        "text": "GLAMOPOLITAN!  We've got ALL the latest tips!  Whether you want to know what the elite are eating, wearing or discussing, Glamopolitan is YOUR magazine!  So pick up a copy today and \"Sizzle Like A Star\"!"
+        "text": "GLAMOPOLITAN!  We've got ALL the latest tips!  Whether you want to know what the elite are eating, wearing or discussing, Glamopolitan is YOUR magazine!  So pick up a copy today and \"Sizzle Like A Star!\""
```

Comment:
These definitely were correct before and shouldn't be switched.

Reply:
That's these and line 268 I presume; what about the rest?